libraries/ESP8266mDNS/src/LEAmDNS.cpp

```diff
@@ -1053,14 +1053,14 @@ const char* MDNSResponder::answerTxts(const MDNSResponder::hMDNSServiceQuery p_h
  */
 bool MDNSResponder::setHostProbeResultCallback(MDNSResponder::MDNSHostProbeFn p_fnCallback) {
 
-	m_HostProbeInformation.m_fnHostProbeResultCallback = p_fnCallback;
+    m_HostProbeInformation.m_fnHostProbeResultCallback = p_fnCallback;
     
     return true;
 }
 
 bool MDNSResponder::setHostProbeResultCallback(MDNSHostProbeFn1 pfn) {
-	using namespace std::placeholders;
-	return setHostProbeResultCallback(std::bind(pfn, std::ref(*this), _1, _2));
+    using namespace std::placeholders;
+    return setHostProbeResultCallback([obj=std::ref(*this), pfn](const char* _1, bool _2) { pfn(obj, _1, _2); });

```

Comment:
What is obj? It's not clear at a glance. Prefer a more descriptive name, e.g. MDNSref or whatever.

Reply:
Compared to bind it's no less clear... *smile* Changing according to signature
```
typedef std::function<void(MDNSResponder& resp,
								   const char* p_pcDomainName,
								   bool p_bProbeResult)> MDNSHostProbeFn1
```